pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[build-system]
+requires = [
+  "setuptools>=42",
+  "setuptools_scm[toml]>=6.2",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+
+[project]
+name = "peek_data"
```

Comment:
Is this the name we intend to keep?

Reply:
I am not sure... I don't have a better name at the moment though.

Reply:
`liveview` and `seelive` are available on `pip`.
I had also named an old project `visens` (for VISualization for Ess Neutron Science) which we can reuse (`visens` means 'seeing' in latin).

Reply:
(`visens` also means `aware` and `a queen bee` in Swedish, and `songs` in Danish. Interesting.)
This dashboard will be one type of visualization but I think they should be separated.

I opened a new issue for naming: #12